docs/designs/rubygems-001.md

```diff
@@ -0,0 +1,547 @@
+# Initial Design for the Rubygems Package Manager
+
+- [Overview](#overview)
+- [Context](#context)
+- [Description](#description)
+    - [Ruby ecosystem overview](#i-ruby-ecosystem-overview)
+    - [Current implementation overview (Cachito)](#ii-overview-of-the-current-implementation-in-cachito)
+    - [Design for the Cachi2 implementation](#iii-design-for-the-implementation-in-cachi2)
+- [Decision](#decision)
+
+## Overview
+
+Design that covers the initial implementation of the Rubygem/Bundler package manager in Cachi2. It takes inspiration from the original implementation done in [Cachito](https://github.com/containerbuildsystem/cachito).
+
+| | |
+|---------|---------|
+|Author |[Michal Šoltis](msoltis@redhat.com) |
+|Co-author |[Bruno Pimentel](bpimente@redhat.com) |
+|Proposed on |August 01, 2024 |
```

Comment:
This was a continuation of the Authors field, which I also struggled to decide if it was worth it or not. What I had in mind was that, although Git tracks info such as commit author and date, and Github tracks PRs author and merge date, if we happen to move this doc around, all of this gets buried under a layer of changes. My reasoning was that this info here would make it easy to reference.

I think "Last updated" is a different info that adds to "proposed date". But in the end, I really don't have a strong feeling for the date fields, so I'd agree to drop it altogether.

Reply:
(I also feel the same for authors... I'd be ok with dropping it)

Reply:
Let's drop them both then, after all this is supposed to be community effort, so whatever design we merge, the whole team/community stands by it. Naturally, when it comes to giving credit, git can help if needed. I think the whole purpose of Authors/Dates etc. fields is for cases where these docs are not tracked in a VCS (is there a git for RFCs, I've never seen one) and so it needs to bear that information otherwise it's automatically public domain :) .

Reply:
I've already had to ask clarification for an author of an old design doc, so that's the main point in having an easy-to-reference author field here, IMO. But we can always dig that up from the commits, so let's just drop these.

Reply:
Dropped both authors and date, resolving this.